tasks/configure-jenkins-users.yml

```diff
@@ -2,8 +2,8 @@
 - name: Ensure Jenkins users exist
   community.general.jenkins_script:
     script: |
-      def userName = '{{ item.key }}'
-      def userPassword = '{{ item.value.password | default("") }}'
+      def userName = '{{ (item | dict2items)[0].key }}'
+      def userPassword = '{{ (item | dict2items)[0].value.password | default("") }}'
```

Comment:
I'm confused, I thought it would look like 0b7e6215657f85bf0ea243fb48fe0a12d732cb62.

Reply:
Something went wrong during rebase. Sorry about that, fixed in 4c13e78.